Cargo.toml

```diff
@@ -11,8 +11,9 @@ documentation = "https://docs.rs/tracing-glog"
 
 [dependencies]
 tracing = { version = "0.1", default-features = false }
-tracing-subscriber = { version = "0.3.3", features = ["std", "fmt", "registry", "time", "local-time"], default-features = false }
+tracing-subscriber = { version = "0.3.18", features = ["std", "fmt", "chrono", "registry", "time", "local-time", "chrono"], default-features = false }
```

Comment:
Just one `"chrono"` is probably enough. Or none if it's being added by our `chrono` feature?

Reply:
whoops! yeah, removing